### PR TITLE
Incorporate controlled vocabulary JSON as part of ISB

### PR DESCRIPTION
**Pull request by dannymandel**
Per discussion with @qgan7125 and @datadavev, these files should be part of the iSB instance.  @qgan7125 to provide instructions on how they are generated so we may update the README.

### CODE REVIEW

**Comment by qgan7125** [comment]
https://github.com/isamplesorg/isamples_inabox/blob/taxonomy/scripts/taxonomy/create_label_hierarchy.py This is the script created by Sarah to generate JSON files. I will add instructions in the webui. I will wait for Sarah's PR.

**Comment by qgan7125** [comment]
Rebase the current branch based on @hyunssong PR and add the instruction about how to generate controlled vocabulary JSON files. Not sure why there are duplicated files in this PR.

File: pyproject.toml [hunk 1]
```diff
@@ -30,10 +30,15 @@ itsdangerous = "^2.1.2"
 starlette-oauth2-api = "^0.2.6"
 httpx = "^0.20.0"
 
-fasttext = "^0.9.2"
+# fasttext = "^0.9.2"
```
Comment (dannymandel) [inline pyproject.toml:33]: Is fasttext complete gone?  Have we removed the Smithsonian code that uses it?

Comment (dannymandel) [reply pyproject.toml:33]: I don't see it in this PR.  This requirement needs to be reinstated or we need to remove the code that was using that (and that will break the Smithsonian transformer until a replacement is ready).

File: isb_web/static/controlled_vocabulary/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+```
+# Generate material_hierarchy.json
+python scripts/taxonomy/create_hierarchy_json.py https://w3id.org/isample/vocabulary/material/0.9/materialsvocabulary > isb_web/static/controlled_vocabulary/material_hierarchy.json
+
+# Generate sampledFeature_hierarchy.json
+python scripts/taxonomy/create_hierarchy_json.py https://raw.githubusercontent.com/isamplesorg/metadata/develop/src/vocabularies/sampledFeature.ttl > isb_web/static/controlled_vocabulary/sampledFeature_hierarchy.json
+
+# Generate specimenType_hierarchy.json
+python scripts/taxonomy/create_hierarchy_json.py https://raw.githubusercontent.com/isamplesorg/metadata/develop/src/vocabularies/specimenType.ttl > isb_web/static/controlled_vocabulary/specimenType_hierarchy.json
+```
```
Comment (dannymandel) [inline isb_web/static/controlled_vocabulary/README.md:22]: This looks great, @qgan7125!  I don't see a note about where the model is supposed to live.  We should include that somewhere.

**Pull request merged**